VideoLocker/res/layout/row_course_outline_list.xml

```diff
@@ -4,8 +4,8 @@
     android:layout_width="match_parent"
     android:layout_height="64dp"
     android:paddingTop="4dp"
-    android:paddingBottom="4dp"
     android:orientation="vertical"
+    android:background="@drawable/course_list_item_selector"
```

Comment:
You should just be able to specify the color as the `listSelector` attribute on the `ListView`, instead of changing the row background to a custom `selector` drawable.